src/main/java/me/totalfreedom/totalfreedommod/command/Command_gtfo.java

```diff
@@ -95,7 +95,7 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
             }
         }
 
-        //checks if there is CoreProtect loaded and installed , if not it skips the rollback and uses coreprotect directly
+        //checks if there is CoreProtect loaded and installed, if not it skips the rollback and uses CoreProtect directly
```

Comment:
Just putting myself out here, this line doesn't make any sense.
It's basically saying if CoreProtect doesn't exists, it would use CoreProtect?

Reply:
It's actually meant to say if it's not enabled it skipa CoreProtect. I just fixed the grammar lol